### ISSUE TITLE
Storing encrypted browser bookmarks, etc.

### ISSUE THREAD

**Issue by impredicative** [etesync/server#71]
Can a cross-browser extension be provided for Firefox, Chrome, and Chromium, along with a corresponding service, to store encrypted bookmarks? Thanks.

This may actually be a part of a broader cross-browser syncing service which also syncs other browser activities, all encrypted.

**Comment by tasn** [comment]
It should be easy to do using [etebase-js](https://github.com/etesync/etebase-js), and we will happily accept such contributions! If you know anyone who's good at building browser extensions and is willing to help, please send them here.

**Comment by tasn** [comment]
Why did you close it?

**Comment by impredicative** [comment]
I just wasn't sure. I definitely would like the feature, including in the paid service.

**Comment by tasn** [comment]
Yeah, I'd love to have it too. Just no time at the moment to tackle it.

**Comment by freespinn** [comment]
Yes! Encrypted bookmarks! We need that !
